deploy/cluster-ingress-operator.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-ingress-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: cluster-ingress-operator
+  template:
+    metadata:
+      labels:
+        name: cluster-ingress-operator
+    spec:
+      containers:
+        - name: cluster-ingress-operator
+          image: openshift/cluster-ingress-operator
+          ports:
+          - containerPort: 60000
+            name: metrics
+          command:
+          - cluster-ingress-operator
+          imagePullPolicy: Always
+          env:
```

Comment:
Isn't `KUBERNETES_CONFIG` also needed?

Reply:
I haven't needed it in my tests. I just run it inside a openshift environment as the system user.
```
oc create -f deploy/cluster-ingress-operator.yaml
```